[12.0][ADD] hr_expense_tier_validation

ps-tubtim opened the pull request:
This module extends the functionality of Expense Reports to support a tier validation process.

Commit:
Module hr_period

saltonmassally committed:
Migrate hr_period from 7.0 to 8.0

serpentcs-dev1 committed:
hr_period v9

AaronHForgeFlow committed:
Adapt hr_period to date_range module

MiquelRForgeFlow committed:
hr_holidays_notify_employee_manager: Migration to 10.0

MiquelRForgeFlow committed:
add company_id in view

nikul-serpentcs committed:
[MIG] hr_period v10

oca-transbot committed:
OCA Transbot updated translations from Transifex

OCA Transbot updated translations from Transifex

OCA Transbot updated translations from Transifex

AaronHForgeFlow committed:
[UPT]Adapt to date range model

nikul-serpentcs committed:
[IMP] Improved code

AaronHForgeFlow committed:
[UPT]improve test coverage

AaronHForgeFlow committed:
[FIX]on change hr period in payslips

AaronHForgeFlow committed:
[FIX]default period on payslips

tarteo committed:
hr_holidays_notify_employee_manager: Migration to 11.0

[MIG] hr_holidays_notify_employee_manager: Migration to 11.0

oca-transbot committed:
OCA Transbot updated translations from Transifex

oca-travis committed:
Update hr_holidays_notify_employee_manager.pot

oca-travis committed:
[UPD] Update hr_period.pot

MiquelRForgeFlow committed:
[FIX] hr_period: don't fail test in onchange_contract

oca-travis committed:
[UPD] Update hr_period.pot

fshahy committed:
Translated using Weblate (Persian)

Currently translated at 66.7% (4 of 6 strings)

Translation: hr-11.0/hr-11.0-hr_holidays_notify_employee_manager
Translate-URL: https://translation.odoo-community.org/projects/hr-11-0/hr-11-0-hr_holidays_notify_employee_manager/fa/

Commit:
Translated using Weblate (Portuguese (Brazil))

Currently translated at 83.3% (5 of 6 strings)

Translation: hr-11.0/hr-11.0-hr_holidays_notify_employee_manager
Translate-URL: https://translation.odoo-community.org/projects/hr-11-0/hr-11-0-hr_holidays_notify_employee_manager/pt_BR/

oca-travis committed:
Update hr_holidays_notify_employee_manager.pot

gurneyalex committed:
Translated using Weblate (French)

Currently translated at 100.0% (82 of 82 strings)

Translation: hr-10.0/hr-10.0-hr_period
Translate-URL: https://translation.odoo-community.org/projects/hr-10-0/hr-10-0-hr_period/fr/

gurneyalex committed:
Translated using Weblate (French)

Currently translated at 100.0% (6 of 6 strings)

Translation: hr-11.0/hr-11.0-hr_holidays_notify_employee_manager
Translate-URL: https://translation.odoo-community.org/projects/hr-11-0/hr-11-0-hr_holidays_notify_employee_manager/fr/

oca-transbot committed:
Update translation files

Updated by Update PO files to match POT (msgmerge) hook in Weblate.

oca-transbot committed:
Update translation files

Updated by Update PO files to match POT (msgmerge) hook in Weblate.

AdriaGForgeFlow committed:
[12.0][MIG] Migrate module hr_period to v12

AaronHForgeFlow committed:
[FIX]hr_period multicompany

oca-travis committed:
[UPD] Update hr_period.pot

Update translation files

Updated by Update PO files to match POT (msgmerge) hook in Weblate.

nikul-serpentcs committed:
[MIG] hr_holidays_notify_employee_manager v12

Commit:
hr_job_category: Migrate to 11.0

oca-travis committed:
Update hr_job_category.pot

Nikul-Chaudhary committed:
[MIG] hr_job_category v11 to v12

alexey-pelykh committed:
[ADD] hr_holidays_leave_request_wizard: advanced leave request wizard and default leave request editor

Commit:
[11.0][FIX] hr_job_category: fix access group

ernestotejeda committed:
hr_expense_cancel: lets to cancel and correct expenses

This module lets to cancel and correct expenses. It adds a cancel
button on the expense sheet that undo reconciliations and delete
payments and journal entries.

danhvo-novobi committed:
[MIG] hr_expense_cancel: Migration to 12.0

Commit:
[ADD] hr_attendance_modify

kittiu committed:
[12.0][ADD] hr_expense_advance_clearing

OCA-git-bot committed:
[UPD] README.rst

oca-travis committed:
[UPD] Update hr_attendance_modification_tracking.pot

Commit:
Added translation using Weblate (Spanish)

Commit:
Translated using Weblate (Spanish)

Currently translated at 100.0% (1 of 1 strings)

Translation: hr-11.0/hr-11.0-hr_attendance_modification_tracking
Translate-URL: https://translation.odoo-community.org/projects/hr-11-0/hr-11-0-hr_attendance_modification_tracking/es/

kittiu committed:
[12.0][IMP] hr_expense_invoice

Add expense info to invoice info or create/edit

kittiu committed:
[12.0][IMP] hr_expense_invoice

From expense sheet, add action "Create Invoice" from multiple expenses
Change the way reference invoice_id is checked.
- No more onchange invoice_id that set values to expense
- Instead, check amount on expense and invoice during post entry
- Change the way to allow reconcile with > 2 account move lines

OCA-git-bot committed:
[ADD] setup.py

oca-travis committed:
[UPD] Update hr_expense_advance_clearing.pot

oca-transbot committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: hr-12.0/hr-12.0-hr_attendance_reason
Translate-URL: https://translation.odoo-community.org/projects/hr-12-0/hr-12-0-hr_attendance_reason/

Commit:
[12.0][ADD] Migration script from hr_employee_seniority

Commit:
[12.0][FIX] Travis Failing Holidays Leave Type Validity

Commit:
Translated using Weblate (Spanish)

Currently translated at 92.5% (37 of 40 strings)

Translation: hr-12.0/hr-12.0-hr_holidays_public
Translate-URL: https://translation.odoo-community.org/projects/hr-12-0/hr-12-0-hr_holidays_public/es/

Commit:
Added translation using Weblate (Spanish)

Commit:
Translated using Weblate (Spanish)

Currently translated at 100.0% (9 of 9 strings)

Translation: hr-12.0/hr-12.0-hr_employee_service_contract
Translate-URL: https://translation.odoo-community.org/projects/hr-12-0/hr-12-0-hr_employee_service_contract/es/

Commit:
Added translation using Weblate (Spanish)

Commit:
Translated using Weblate (Spanish)

Currently translated at 100.0% (16 of 16 strings)

Translation: hr-12.0/hr-12.0-hr_employee_service
Translate-URL: https://translation.odoo-community.org/projects/hr-12-0/hr-12-0-hr_employee_service/es/

Commit:
Added translation using Weblate (Spanish)

Commit:
Translated using Weblate (Spanish)

Currently translated at 94.1% (16 of 17 strings)

Translation: hr-12.0/hr-12.0-hr_employee_calendar_planning
Translate-URL: https://translation.odoo-community.org/projects/hr-12-0/hr-12-0-hr_employee_calendar_planning/es/

Saran440 committed:
[12.0][MIG] hr_attendance_modification_tracking

alexey-pelykh committed:
[FIX] Allow reading skills by everyone

OCA-git-bot committed:
hr_skill 12.0.1.1.0

marylla committed:
Translated using Weblate (German)

Currently translated at 100.0% (4 of 4 strings)

Translation: hr-12.0/hr-12.0-hr_holidays_leave_auto_approve
Translate-URL: https://translation.odoo-community.org/projects/hr-12-0/hr-12-0-hr_holidays_leave_auto_approve/de/

marylla committed:
Translated using Weblate (German)

Currently translated at 100.0% (33 of 33 strings)

Translation: hr-12.0/hr-12.0-hr_skill
Translate-URL: https://translation.odoo-community.org/projects/hr-12-0/hr-12-0-hr_skill/de/

marylla committed:
Translated using Weblate (German)

Currently translated at 81.2% (13 of 16 strings)

Translation: hr-12.0/hr-12.0-hr_employee_id
Translate-URL: https://translation.odoo-community.org/projects/hr-12-0/hr-12-0-hr_employee_id/de/

marylla committed:
Translated using Weblate (German)

Currently translated at 100.0% (4 of 4 strings)

Translation: hr-12.0/hr-12.0-hr_employee_firstname
Translate-URL: https://translation.odoo-community.org/projects/hr-12-0/hr-12-0-hr_employee_firstname/de/

marylla committed:
Translated using Weblate (German)

Currently translated at 100.0% (2 of 2 strings)

Translation: hr-12.0/hr-12.0-hr_employee_birth_name
Translate-URL: https://translation.odoo-community.org/projects/hr-12-0/hr-12-0-hr_employee_birth_name/de/

marylla committed:
Translated using Weblate (German)

Currently translated at 100.0% (17 of 17 strings)

Translation: hr-12.0/hr-12.0-hr_employee_social_media
Translate-URL: https://translation.odoo-community.org/projects/hr-12-0/hr-12-0-hr_employee_social_media/de/

marylla committed:
Translated using Weblate (German)

Currently translated at 100.0% (19 of 19 strings)

Translation: hr-12.0/hr-12.0-hr_contract_multi_job
Translate-URL: https://translation.odoo-community.org/projects/hr-12-0/hr-12-0-hr_contract_multi_job/de/

marylla committed:
Translated using Weblate (German)

Currently translated at 92.5% (37 of 40 strings)

Translation: hr-12.0/hr-12.0-hr_holidays_public
Translate-URL: https://translation.odoo-community.org/projects/hr-12-0/hr-12-0-hr_holidays_public/de/

alexey-pelykh committed:
[FIX] hr_holidays_accrual_advanced: tests and more logs

oca-travis committed:
[UPD] Update hr_holidays_accrual_advanced.pot

OCA-git-bot committed:
hr_holidays_accrual_advanced 12.0.1.0.1

oca-travis committed:
[UPD] Update hr_attendance_modification_tracking.pot

OCA-git-bot committed:
[UPD] README.rst

OCA-git-bot committed:
[ADD] setup.py

marylla committed:
Translated using Weblate (German)

Currently translated at 100.0% (4 of 4 strings)

Translation: hr-12.0/hr-12.0-hr_holidays_settings
Translate-URL: https://translation.odoo-community.org/projects/hr-12-0/hr-12-0-hr_holidays_settings/de/

marylla committed:
Translated using Weblate (German)

Currently translated at 100.0% (40 of 40 strings)

Translation: hr-12.0/hr-12.0-hr_holidays_public
Translate-URL: https://translation.odoo-community.org/projects/hr-12-0/hr-12-0-hr_holidays_public/de/

OCA-git-bot committed:
[UPD] README.rst

OCA-git-bot committed:
[ADD] setup.py

oca-travis committed:
[UPD] Update hr_holidays_notify_employee_manager.pot

marylla committed:
Translated using Weblate (German)

Currently translated at 100.0% (16 of 16 strings)

Translation: hr-12.0/hr-12.0-hr_employee_id
Translate-URL: https://translation.odoo-community.org/projects/hr-12-0/hr-12-0-hr_employee_id/de/

marylla committed:
Translated using Weblate (German)

Currently translated at 100.0% (26 of 26 strings)

Translation: hr-12.0/hr-12.0-hr_attendance_report_theoretical_time
Translate-URL: https://translation.odoo-community.org/projects/hr-12-0/hr-12-0-hr_attendance_report_theoretical_time/de/

marylla committed:
Translated using Weblate (German)

Currently translated at 100.0% (4 of 4 strings)

Translation: hr-12.0/hr-12.0-hr_expense_invoice
Translate-URL: https://translation.odoo-community.org/projects/hr-12-0/hr-12-0-hr_expense_invoice/de/

marylla committed:
Translated using Weblate (German)

Currently translated at 100.0% (48 of 48 strings)

Translation: hr-12.0/hr-12.0-hr_experience
Translate-URL: https://translation.odoo-community.org/projects/hr-12-0/hr-12-0-hr_experience/de/

marylla committed:
Translated using Weblate (German)

Currently translated at 100.0% (6 of 6 strings)

Translation: hr-12.0/hr-12.0-hr_holidays_notify_employee_manager
Translate-URL: https://translation.odoo-community.org/projects/hr-12-0/hr-12-0-hr_holidays_notify_employee_manager/de/

marylla committed:
Translated using Weblate (German)

Currently translated at 100.0% (3 of 3 strings)

Translation: hr-12.0/hr-12.0-hr_payroll_cancel
Translate-URL: https://translation.odoo-community.org/projects/hr-12-0/hr-12-0-hr_payroll_cancel/de/

dalonsod committed:
Added translation using Weblate (Spanish)

dalonsod committed:
Translated using Weblate (Spanish)

Currently translated at 100.0% (46 of 46 strings)

Translation: hr-12.0/hr-12.0-hr_employee_health
Translate-URL: https://translation.odoo-community.org/projects/hr-12-0/hr-12-0-hr_employee_health/es/

alexey-pelykh committed:
[FIX] hr_holidays_settings: duplicate Leave Types menuitem

alexey-pelykh committed:
[FIX] hr_holidays_accrual_advanced: wizard crash

oca-transbot committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: hr-12.0/hr-12.0-hr_holidays_accrual_advanced
Translate-URL: https://translation.odoo-community.org/projects/hr-12-0/hr-12-0-hr_holidays_accrual_advanced/

oca-transbot committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: hr-12.0/hr-12.0-hr_holidays_notify_employee_manager
Translate-URL: https://translation.odoo-community.org/projects/hr-12-0/hr-12-0-hr_holidays_notify_employee_manager/

oca-transbot committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: hr-12.0/hr-12.0-hr_attendance_modification_tracking
Translate-URL: https://translation.odoo-community.org/projects/hr-12-0/hr-12-0-hr_attendance_modification_tracking/

oca-travis committed:
[UPD] Update hr_holidays_leave_request_wizard.pot

OCA-git-bot committed:
[ADD] setup.py

OCA-git-bot committed:
[ADD] icon.png

Commit:
Translated using Weblate (Spanish)

Currently translated at 100.0% (40 of 40 strings)

Translation: hr-12.0/hr-12.0-hr_holidays_public
Translate-URL: https://translation.odoo-community.org/projects/hr-12-0/hr-12-0-hr_holidays_public/es/

Commit:
Translated using Weblate (Spanish)

Currently translated at 100.0% (4 of 4 strings)

Translation: hr-12.0/hr-12.0-hr_holidays_leave_auto_approve
Translate-URL: https://translation.odoo-community.org/projects/hr-12-0/hr-12-0-hr_holidays_leave_auto_approve/es/

Commit:
Translated using Weblate (Spanish)

Currently translated at 100.0% (26 of 26 strings)

Translation: hr-12.0/hr-12.0-hr_attendance_report_theoretical_time
Translate-URL: https://translation.odoo-community.org/projects/hr-12-0/hr-12-0-hr_attendance_report_theoretical_time/es/

Commit:
Translated using Weblate (Spanish)

Currently translated at 100.0% (4 of 4 strings)

Translation: hr-12.0/hr-12.0-hr_employee_firstname
Translate-URL: https://translation.odoo-community.org/projects/hr-12-0/hr-12-0-hr_employee_firstname/es/

Commit:
Translated using Weblate (Spanish)

Currently translated at 100.0% (4 of 4 strings)

Translation: hr-12.0/hr-12.0-hr_expense_invoice
Translate-URL: https://translation.odoo-community.org/projects/hr-12-0/hr-12-0-hr_expense_invoice/es/

Commit:
Translated using Weblate (Spanish)

Currently translated at 100.0% (5 of 5 strings)

Translation: hr-12.0/hr-12.0-hr_employee_phone_extension
Translate-URL: https://translation.odoo-community.org/projects/hr-12-0/hr-12-0-hr_employee_phone_extension/es/

Commit:
Translated using Weblate (Spanish)

Currently translated at 100.0% (6 of 6 strings)

Translation: hr-12.0/hr-12.0-hr_holidays_notify_employee_manager
Translate-URL: https://translation.odoo-community.org/projects/hr-12-0/hr-12-0-hr_holidays_notify_employee_manager/es/

Commit:
Translated using Weblate (Spanish)

Currently translated at 100.0% (2 of 2 strings)

Translation: hr-12.0/hr-12.0-hr_employee_birth_name
Translate-URL: https://translation.odoo-community.org/projects/hr-12-0/hr-12-0-hr_employee_birth_name/es/

Commit:
Translated using Weblate (Spanish)

Currently translated at 100.0% (4 of 4 strings)

Translation: hr-12.0/hr-12.0-hr_attendance_modification_tracking
Translate-URL: https://translation.odoo-community.org/projects/hr-12-0/hr-12-0-hr_attendance_modification_tracking/es/

Commit:
Translated using Weblate (Spanish)

Currently translated at 100.0% (17 of 17 strings)

Translation: hr-12.0/hr-12.0-hr_employee_calendar_planning
Translate-URL: https://translation.odoo-community.org/projects/hr-12-0/hr-12-0-hr_employee_calendar_planning/es/

Commit:
Translated using Weblate (Spanish)

Currently translated at 51.5% (17 of 33 strings)

Translation: hr-12.0/hr-12.0-hr_skill
Translate-URL: https://translation.odoo-community.org/projects/hr-12-0/hr-12-0-hr_skill/es/

alexey-pelykh committed:
[IMP] hr_employee_service: support _get_date_start_work

mvasquez13 committed:
[FIX]  hr_employee_service: Don't try to access field service_duration for getting label

It will fail if no permissions

alexey-pelykh committed:
[FIX] hr_skill: fix permissions

oca-travis committed:
[UPD] Update hr_employee_service.pot

OCA-git-bot committed:
hr_skill 12.0.1.2.0

Commit:
Added translation using Weblate (Spanish)

OCA-git-bot committed:
hr_employee_service 12.0.1.0.1

oca-transbot committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: hr-12.0/hr-12.0-hr_employee_service
Translate-URL: https://translation.odoo-community.org/projects/hr-12-0/hr-12-0-hr_employee_service/

OCA-git-bot committed:
[UPD] README.rst

alexey-pelykh committed:
[FIX] hr_holidays_notify_employee_manager: tests

OCA-git-bot committed:
[UPD] README.rst

oca-travis committed:
[UPD] Update hr_holidays_settings.pot

OCA-git-bot committed:
[UPD] README.rst

OCA-git-bot committed:
hr_holidays_settings 12.0.1.1.0

oca-travis committed:
[UPD] Update hr_period.pot

OCA-git-bot committed:
[UPD] README.rst

OCA-git-bot committed:
[ADD] setup.py

Commit:
Added translation using Weblate (Spanish)

Commit:
Translated using Weblate (Spanish)

Currently translated at 100.0% (7 of 7 strings)

Translation: hr-12.0/hr-12.0-hr_attendance_rfid
Translate-URL: https://translation.odoo-community.org/projects/hr-12-0/hr-12-0-hr_attendance_rfid/es/

Commit:
Translated using Weblate (Spanish)

Currently translated at 4.4% (4 of 91 strings)

Translation: hr-12.0/hr-12.0-hr_period
Translate-URL: https://translation.odoo-community.org/projects/hr-12-0/hr-12-0-hr_period/es/

Commit:
Translated using Weblate (Spanish)

Currently translated at 7.7% (7 of 91 strings)

Translation: hr-12.0/hr-12.0-hr_period
Translate-URL: https://translation.odoo-community.org/projects/hr-12-0/hr-12-0-hr_period/es/

oca-travis committed:
[UPD] Update hr_job_category.pot

OCA-git-bot committed:
[UPD] README.rst

OCA-git-bot committed:
[ADD] setup.py

oca-transbot committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: hr-12.0/hr-12.0-hr_job_category
Translate-URL: https://translation.odoo-community.org/projects/hr-12-0/hr-12-0-hr_job_category/

Commit:
[12.0][IMP] Add migration script hr_family -> hr_employee_relative

oca-travis committed:
[UPD] Update hr_expense_cancel.pot

OCA-git-bot committed:
[UPD] README.rst

OCA-git-bot committed:
[ADD] setup.py

OCA-git-bot committed:
[ADD] icon.png

HviorForgeFlow committed:
[12.0][FIX] hr_holidays_notify_employee_manager: Add readonly False on leave_notify_manager option

marylla committed:
Translated using Weblate (German)

Currently translated at 100.0% (4 of 4 strings)

Translation: hr-12.0/hr-12.0-hr_job_category
Translate-URL: https://translation.odoo-community.org/projects/hr-12-0/hr-12-0-hr_job_category/de/

marylla committed:
Translated using Weblate (German)

Currently translated at 100.0% (6 of 6 strings)

Translation: hr-12.0/hr-12.0-hr_holidays_notify_employee_manager
Translate-URL: https://translation.odoo-community.org/projects/hr-12-0/hr-12-0-hr_holidays_notify_employee_manager/de/

OCA-git-bot committed:
hr_holidays_notify_employee_manager 12.0.1.1.0

oca-transbot committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: hr-12.0/hr-12.0-hr_expense_cancel
Translate-URL: https://translation.odoo-community.org/projects/hr-12-0/hr-12-0-hr_expense_cancel/

oca-transbot committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: hr-12.0/hr-12.0-hr_period
Translate-URL: https://translation.odoo-community.org/projects/hr-12-0/hr-12-0-hr_period/

oca-transbot committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: hr-12.0/hr-12.0-hr_holidays_settings
Translate-URL: https://translation.odoo-community.org/projects/hr-12-0/hr-12-0-hr_holidays_settings/

Commit:
Translated using Weblate (Portuguese (Brazil))

Currently translated at 100.0% (6 of 6 strings)

Translation: hr-12.0/hr-12.0-hr_holidays_notify_employee_manager
Translate-URL: https://translation.odoo-community.org/projects/hr-12-0/hr-12-0-hr_holidays_notify_employee_manager/pt_BR/

sbidoul committed:
Deleted translation using Weblate (Spanish)

Commit:
[12.0][FIX] hr_employee_service: Test 3

Commit:
Translated using Weblate (Portuguese (Brazil))

Currently translated at 100.0% (5 of 5 strings)

Translation: hr-12.0/hr-12.0-hr_employee_phone_extension
Translate-URL: https://translation.odoo-community.org/projects/hr-12-0/hr-12-0-hr_employee_phone_extension/pt_BR/

Commit:
Translated using Weblate (Portuguese (Brazil))

Currently translated at 100.0% (4 of 4 strings)

Translation: hr-12.0/hr-12.0-hr_employee_firstname
Translate-URL: https://translation.odoo-community.org/projects/hr-12-0/hr-12-0-hr_employee_firstname/pt_BR/

Commit:
Translated using Weblate (Portuguese (Brazil))

Currently translated at 100.0% (4 of 4 strings)

Translation: hr-12.0/hr-12.0-hr_job_category
Translate-URL: https://translation.odoo-community.org/projects/hr-12-0/hr-12-0-hr_job_category/pt_BR/

Commit:
Translated using Weblate (Portuguese (Brazil))

Currently translated at 100.0% (33 of 33 strings)

Translation: hr-12.0/hr-12.0-hr_skill
Translate-URL: https://translation.odoo-community.org/projects/hr-12-0/hr-12-0-hr_skill/pt_BR/

Commit:
Added translation using Weblate (Spanish)

Commit:
[12.0][IMP] Add phone number and job to hr_employee_relative

oca-travis committed:
[UPD] Update hr_employee_relative.pot

OCA-git-bot committed:
hr_employee_relative 12.0.1.1.0

oca-transbot committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: hr-12.0/hr-12.0-hr_employee_relative
Translate-URL: https://translation.odoo-community.org/projects/hr-12-0/hr-12-0-hr_employee_relative/

Commit:
[12.0][IMP] hr_attendance_report_theoretical_time: Add filter wizard

damdam-s committed:
FIX: fix leave number days when requested across years and public holidays are defined for each year

OCA-git-bot committed:
hr_holidays_public 12.0.1.0.1

flotho committed:
Fix bad syntax in query

the query is not correct because of semicolon

Commit:
[12.0][IMP] hr_employee_social_media: Add tests

oca-travis committed:
[UPD] Update hr_attendance_report_theoretical_time.pot

OCA-git-bot committed:
hr_attendance_report_theoretical_time 12.0.1.1.0

oca-transbot committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: hr-12.0/hr-12.0-hr_attendance_report_theoretical_time
Translate-URL: https://translation.odoo-community.org/projects/hr-12-0/hr-12-0-hr_attendance_report_theoretical_time/

Commit:
[12.0][FIX] hr_holiday_public: Fix method return for concatenation

ps-tubtim committed:
[12.0][ADD] hr_expense_tier_validation

kittiu commented on hr_expense_tier_validation/__manifest__.py:
```diff
@@ -0,0 +1,20 @@
+    'data': [
+        'views/hr_expense_sheet_view.xml',
+    ],
+    'installable': True,
```
Please add yourself as maintainers

oca-travis committed:
[UPD] Update hr_expense_invoice.pot

LoisRForgeFlow commented:
Could you please rebase your branch? that would fix runbot.

OCA-git-bot committed:
[UPD] README.rst

OCA-git-bot committed:
hr_expense_invoice 12.0.1.3.0

oca-transbot committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: hr-12.0/hr-12.0-hr_expense_invoice
Translate-URL: https://translation.odoo-community.org/projects/hr-12-0/hr-12-0-hr_expense_invoice/

LoisRForgeFlow commented:
To rebase properly you also need to fetch the oca remote first, if you navigate your branch you will see that is not up-to-date (your most recent commit appart from this PR's is from June).

LoisRForgeFlow commented:
Superseded by https://github.com/OCA/hr/pull/696
